Orange/widgets/visualize/tests/test_owpythagoreanforest.py

```diff
@@ -131,13 +134,10 @@ def test_changing_size_adjustment_changes_sizes(self):
         w = self.widget
         self.send_signal(w.Inputs.random_forest, self.titanic)
         squares = []
-        # We have to get the same tree with an index on the grid items since
-        # the tree objects are deleted and recreated with every invalidation
-        tree_index = w.grid_items.index(random.choice(self.get_grid_items()))
+        tree = self._pick_random_tree()
```

Comment:
I think we're trying to avoid randomness in tests if possible. Ask @markotoplak, please.